src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareAsString = (a, b) => {
+    const aToString = String(a);
+    const bToString = String(b);
+
+    if (aToString > bToString) {
+      return 1;
+    } else if (aToString === bToString) {
+      return 0;
+    } else if (aToString < bToString) {
+      return -1;
+    }
+  };
+
```

Comment:
if you return value, no need else here, and also no need to compare  aToString === bToString , just use return 0 in the end